src/components/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import {Route, Router, Routes} from "react-router-dom";
+import Header from './header/Header'
+import ManageLoginData from './ManageLoginData';
+import HomePage from './HomePage';
+import ManageRegisterData from "./ManageRegisterData";
+import PageNotFound from './PageNotFound'
+const App = () => {
```

Comment:
whitespace between function/component declaration and the last import line please